src/lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.tsx

```diff
@@ -38,7 +42,13 @@ export function ClimateSettingScheduleForm({
 function Content({
   onBack,
 }: Omit<ClimateSettingScheduleFormProps, 'className'>): JSX.Element {
-  const { control } = useForm()
+  const { control, watch } = useForm()
+
+  const deviceId = watch('deviceId', null)
+
+  const { device } = useDevice({
+    device_id: deviceId,
+  })
```

Comment:
I'd like to let you override the react query params so I can do something like:
``` 
const { device } = useDevice({
    device_id: deviceId,
  },
	{enabled: Boolean(deviceId)}
)
```

Reply:
I think we can just split each step into its own component to handle this.

Reply:
great idea!

Reply:
So one hiccup, all the other steps do have the same heading, I did split it out though and it seems cleaner. 

I can put `<ContentHeader/>` above  `<Content/>`, that seems appropriate 😅

Reply:
https://github.com/seamapi/react/pull/460/commits/68c485e177f171557e794df536a9edc09be21c3a